components/popover/index.tsx

```diff
@@ -11,7 +12,7 @@ export interface PopoverProps extends AbstractTooltipProps {
 export default class Popover extends React.Component<PopoverProps, {}> {
   static defaultProps = {
     prefixCls: 'ant-popover',
-    placement: 'top',
+    placement: 'top' as TooltipPlacement,
     transitionName: 'zoom-big',
     trigger: 'hover',
```

Comment:
this one should also be `'hover' | 'focus' | 'click'` since the type it should have